src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -472,19 +472,23 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<MakeDir Directories="$(_AndroidStampDirectory)" Condition=" !Exists('$(_AndroidStampDirectory)') " />
 	<MakeDir Directories="$(_AndroidIntermediateDesignTimeBuildDirectory)" Condition=" !Exists ('$(_AndroidIntermediateDesignTimeBuildDirectory)') " />
 </Target>
+
+<ItemGroup>
+	<AndroidCustomMetaDataForReferences Include="@(_AndroidAssemblySkipCases)" />
+</ItemGroup>
 	
 <Target Name="_AddAndroidCustomMetaData">
   <AppendCustomMetadataToItemGroup
       Condition="'$(AndroidApplication)' != '' And $(AndroidApplication)"
       Inputs="@(ReferencePath)"
-      MetaDataItems="@(_AndroidAssemblySkipCases)"
+      MetaDataItems="@(AndroidCustomMetaData)"
```

Comment:
Should this be `AndroidCustomMetaData` or `AndroidCustomMetaDataForReferences`? Is this similarity just a coincidence?

Reply:
🤦‍♂️  Fixed